src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -513,8 +513,8 @@ private function renderCell(array $row, int $column, string $cellFormat): string
         }
 
         // str_pad won't work properly with multi-byte strings, we need to fix the padding
-        if (false !== $encoding = mb_detect_encoding($cell, null, true)) {
-            $width += \strlen($cell) - mb_strwidth($cell, $encoding);
+        if (null !== $cell && false !== $encoding = mb_detect_encoding((string) $cell, null, true)) {
```

Comment:
`$cell` can't be null here
if the cast to string is needed, let's make it happen once

Reply:
Done.